api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -159,7 +159,11 @@ public void setReplicas(int replicas) {
         this.replicas = replicas;
     }
 
-    @Description("The docker image for the pods. The default value depends on the configured `Kafka.spec.kafka.version`.")
+    @Description("The Docker image used for generating Kafka pods. "
+        + "If the property is not set, the default Kafka image version is determined based on the `version` configuration. "  
+        + "If a version is not configured, the image name corresponds to the default version specified in the Cluster Operator configuration. "
+        + "Changing the Kafka image version does not automatically update the image versions for other components, such as Kafka Exporter. " 
+        + "Specific image versions for these components must be set as needed.")
```

Comment:
The other components such as Kafka Exporter are not version dependent. So I do not think the last sentence makes sense as there is no real need to set those.